### ISSUE TITLE
showIf/hideIf for a whole LayoutStart possible?

### ISSUE THREAD

**Issue by hungin** [TylerTemp/SaintsField#100]
Is it somehow possible show or hide a whole LayoutStart Group, at the moment I can only hide the vars in the group leaving a empty group.

**Comment by TylerTemp** [comment]
Hi,

this is a in-plan feature, see [#73](https://github.com/TylerTemp/SaintsField/issues/73). For more plans, see [the current milestone](https://github.com/TylerTemp/SaintsField/milestone/7)

**Comment by TylerTemp** [comment]
Hi,

this feature has been implemented in [3.7.0](https://github.com/TylerTemp/SaintsField/releases/tag/3.7.0)

```csharp
using SaintsField.Playa;

public bool editableMain;
public bool visibleMain;

[LayoutEnableIf(nameof(editableMain))]
[LayoutShowIf(nameof(visibleMain))]
[LayoutStart("Main", ELayout.FoldoutBox)]
public bool editable1;
public bool visible1;

[LayoutEnableIf(nameof(editable1))]
[LayoutShowIf(nameof(visible1))]
[LayoutStart("./1", ELayout.FoldoutBox, marginBottom: 10)]
public int int1;
public string string1;

[LayoutStart("..", ELayout.FoldoutBox)]
public bool editable2;
public bool visible2;


[LayoutEnableIf(nameof(editable2))]
[LayoutShowIf(nameof(visible2))]
[LayoutStart("./2", ELayout.FoldoutBox)]
public int int2;
public string string2;
```

[![video](https://github.com/user-attachments/assets/f437ebe4-b4f0-4d3e-be8b-646dbdb74eca)](https://github.com/user-attachments/assets/fac5fce5-6458-4853-893c-23fa50f84872)
